test/loki_service.cc

```diff
@@ -90,14 +90,6 @@ const std::vector<http_request_t> valhalla_requests{
     http_request_t(
         GET,
         R"(/route?json={"locations":[{"lon":0,"lat":0},{"lon":0,"lat":0}],"costing":"pedestrian","exclude_locations":[{"lon":0,"lat":0}]})"),
-    http_request_t(
-        POST,
-        "/trace_attributes",
-        R"({"shape":[{"lat":37.8077440,"lon":-122.4197010},{"lat":37.8077440,"lon":-122.4197560},{"lat":37.8077450,"lon":-122.4198180}],"shape_match":"map_snap","best_paths":0,"costing":"pedestrian","directions_options":{"units":"miles"}})"),
-    http_request_t(
-        POST,
-        "/trace_attributes",
-        R"({"shape":[{"lat":37.8077440,"lon":-122.4197010},{"lat":37.8077440,"lon":-122.4197560},{"lat":37.8077450,"lon":-122.4198180}],"shape_match":"map_snap","best_paths":5,"costing":"pedestrian","directions_options":{"units":"miles"}})"),
```

Comment:
instead of sending an error about asking for too many alternates in map matching i changed it to do what we do for regular routing which is to cap it. in the end this isnt a breaking change as the user wasnt getting back a usable response in the first place when asking for too few or too many routes.

Reply:
good as it's more inline with the general philosophy. I still think these implicit request parameter changes should be communicated in the response, e.g. #3099 . some day.. :)